### PR TITLE
Audio folder normalization

### PR DESCRIPTION
**Pull request by Itee**
Next to #10637, #10648, #10658, #10660, #10661, #10662, #10663

Plus apply MrDoob eslint style

### CODE REVIEW

File: src/audio/Audio.js [hunk 1]
```diff
@@ -56,7 +56,11 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 		this.buffer = audioBuffer;
 		this.sourceType = 'buffer';
 
-		if ( this.autoplay ) this.play();
```
Comment (mrdoob) [inline src/audio/Audio.js]: I actually prefer how this line was already. That's one of the issues I have with enforcing the eslint stuff...

Comment (TristanVALCKE) [inline src/audio/Audio.js]: Ok i will take care about that ! An other things that can cause trouble is spacement for array of vertices/indexes/matrix. Are you agree if i add some "disable comment" for eslint rules in this cases ?

Comment (mrdoob) [inline src/audio/Audio.js]: Yeah, lets see how that results.

**Comment by Mugen87** [comment]
Most of these changes were covered by #12156 and subsequent PRs. I think we can close this series or PRs.

**Pull request closed without merging**